packages/discord-snippet/0.1.0/package.yml

```diff
@@ -0,0 +1,110 @@
+# espanso configuration file
+
+# This is the default configuration file, change it as you like it
+# You can refer to the official documentation:
+# https://espanso.org/docs/
+
+# Matches are the substitution rules, when you type the "trigger" string
+# it gets replaced by the "replace" string.
+name: discord-snippet
+parent: default
+
+matches:
+  ### ----------------
+  ### Text Formatting
+  ### ----------------
+
+  # Discord Italic
+- trigger: ":dita"
+  replace: "_$|$_"
+
+  # Discord Bold
+- trigger: ":dbo"
+  replace: "**$|$**"
+
+  # Discord Underline
+- trigger: ":dun"
+  replace: "__$|$__"
+
+  # Discord Bold Italic
+- trigger: ":ditb"
+  replace: "***$|$***"
+
+  # Discord Underline Italic
+- trigger: ":duit"
+  replace: "__*$|$*__"
+
+  # Discord Underline Bold
+- trigger: ":dubo"
+  replace: "__**$|$**__"
+
+  # Discord Underline Bold Italic
+- trigger: ":dubi"
+  replace: "__***$|$***__"
+
+  # Discord Strikethrough
+- trigger: ":dst"
+  replace: "~~$|$~~"
+
+   # Discord Header 1
+- trigger: ":dh1"
+  replace: "# $|$"
+
+  # Discord Header 2
+- trigger: ":dh2"
+  replace: "## $|$"
+
+  # Discord Header 3
+- trigger: ":dh3"
+  replace: "### $|$"
+
+
+  ### ----------------
+  ### Text Blocks
+  ### ----------------
+
+  # Discord Code block
+- trigger: ":dco"
+  replace: "`$|$`"
+
+  # Discord Multi line Code Block
+- trigger: ":dmco"
+  replace: "```$|$```"
+
+  # Discord Spoiler
+- trigger: ":dsp"
+  replace: "||$|$||"
+
+  # Discord Quote
+- trigger: ":dqu"
+  replace: "> $|$"
+
+  # Discord Block Quote
+- trigger: ":dbq"
+  replace: ">>> $|$"
+
+  ### ----------------
+  ### Links and Mentions
+  ### ----------------
+
+  # Discord Mention
+- trigger: ":dme"
+  replace: "@$|$"
+
+  # Discord Channel Mention
+- trigger: ":dch"
+  replace: "#$|$"
+
+  # Discord Role Mention
+- trigger: ":dro"
+  replace: "@&$|$"
+
+  # Discord Masked Link
+- trigger: ":dli"
```

Comment:
Any reason for not using:
```yml
- trigger: ":dli"
  replace: "[$|$]({{clipboard_content}})"
  vars:
  - name: clipboard_content
    type: clipboard
```
which will work in all OSs?

Reply:
Hi,
I didn't know this existed, I just started using yml, thanks for the tip.
I tested it and it works.

Reply:
Thank you.

See https://espanso.org/docs/matches/extensions/#clipboard-extension